float32 vehSpeed
float64 pointX
float64 pointY
float64 pointHA
int32 lane_type
string caption
path_point_action[] actions


